app/views/landmarks/show.html.erb

```diff
@@ -1,5 +1,22 @@
 <h3><%= @landmark.name %></h3>
-  <%= image_tag("#{@landmark.image}") %>
+<%= image_tag("#{@landmark.image}", height: '250', width: '250') %>
 <%= @landmark.location %>
 <%= @landmark.description %>
-   
+<br><br>
+<%= render './reviews/form' %>
```

Comment:
You shouldn't need the period at the beginning of this relative path. Are you having trouble getting it to work without it?